src/explore-education-statistics-common/src/modules/table-tool/components/LocationFiltersForm.tsx

```diff
@@ -170,10 +170,10 @@ const LocationFiltersForm = (props: Props & InjectedWizardProps) => {
                     shouldCollapse
                   >
                     {sortBy(levelOptions, ['label']).map(level => (
-                      <React.Fragment key={level.value}>
+                      <div key={level.value}>
```

Comment:
Think these should be `li`s. Think this current change would result in non-semantic markup that looks like:

```html
<ul>
   <div>Location 1</div>
   <div>Location 2</div>
</ul>
```

Reply:
You are right -- in fact I actually made that change at one point